addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -171,6 +171,11 @@ $-o-theme-text-colors: null;
     @extend %-bg-black;
 }
 
+.bg-transparent{
+    --bg-opacity: 0;
+    @include o-print-color($black, background-color, bg-opacity);
```

Comment:
@chgo-odoo , thanks.

Can we actually `@extend %-bg-black`?
Note that the comment on top (L136) should be updated accordingly ;)